ProductionSystem/DB/ProductionDB.sql

```diff
@@ -44,8 +44,8 @@ CREATE TABLE ProductionSteps(
     Plugin CHAR(32) DEFAULT 'None',
     AgentType CHAR(32) DEFAULT 'Manual',
     GroupSize INT NOT NULL DEFAULT 1,
-    InputQuery VARCHAR(255) DEFAULT 'None',
-    OutputQuery VARCHAR(255) DEFAULT 'None',
+    InputQuery LONGBLOB,
+    OutputQuery LONGBLOB,
```

Comment:
No more `DEFAULT`s?

Reply:
It seems that LONGBLOB cannot have a default value. The test previous to the last commit failed for this reason.

Reply:
OK. BTW this change should be advertised, please do it in https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v7r1